lib/intercepted_request_router.js

```diff
@@ -31,6 +31,9 @@ class InterceptedRequestRouter {
     this.interceptors = interceptors
 
     this.socket = new Socket(options)
+    if (options.timeout) {
+      this.socket.setTimeout(options.timeout)
```

Comment:
Could you add a link here to the Node API docs and to this PR?

Reply:
Fixed :D